tests/src/Traits/ResponseModelTestTrait.php

```diff
@@ -124,17 +132,32 @@ public function createResponseValidationErrors(array $data = []): ValidationErro
      */
     public function createResponseFile(array $data = []): File
     {
-        return (new File())
+        /**
+         * @var \OpenEuropa\CdtClient\Model\Response\LinkCollection $linkCollection
+         */
+        $linkCollection = $this->createResponseObjectList(
+            $data['links'] ?? null,
+            [$this, 'createResponseLink'],
+            LinkCollection::class,
+            'first',
+        );
+        $file = (new File())
             ->setFilename($data['fileName'] ?? 'test.xml')
-            ->setSourceLanguage($data['sourceLanguage'] ?? 'EN')
-            ->setTargetLanguage($data['targetLanguage'] ?? 'FR')
-            ->setSourceDocument($data['sourceDocument'] ?? 'test.xml')
             ->setIsPrivate($data['isPrivate'] ?? false)
-            ->setLinks($this->createResponseObjectList(
-                $data['links'] ?? null,
-                [$this, 'createResponseLink'],
-                'first',
-            ));
+            ->setLinks($linkCollection);
+
+        // Set optional parameters.
```

Comment:
This needed a slight refactor, to make these parameters truly optional. Otherwise, empty values were replaced by default ones.